requirements.txt

```diff
@@ -0,0 +1,15 @@
+django==2.2.8
+mysqlclient==1.4.5
+whitenoise==4.1.1
+pillow==6.2.0
+django-markdownx==2.0.27
+gunicorn==19.9.0
+fabric==2.4.0
+social-auth-app-django==3.1.0
+django-autocomplete-light==3.5.1
+
+pre-commit==1.14.4
```

Comment:
I wouldn't mix the dev and prod requirements together. Often the dev ones are in a `requirements-dev.txt` instead.

It's also usually useful to have a place where they're pinned (like they are here) with all dependencies pinned too, and then have a place for the actual looser requirements directly needed for the app (django for instance, but not specifying a version unless there's compatibility problems), this makes it much easier to be able to upgrade and know why things are pinned.

Reply:
Separated out prod and dev dependencies. Not sure I'm going to follow through with the pinned/unpinned version of our requirements at the moment, adding it as a TODO in the files.